adapters/brightroll/brightroll.go

```diff
@@ -37,13 +65,21 @@ func (a *BrightrollAdapter) MakeRequests(request *openrtb.BidRequest) ([]*adapte
 				bannerCopy.W = &(firstFormat.W)
 				bannerCopy.H = &(firstFormat.H)
 			}
+			if brightrollExt.Publisher == "adthrive" {
+				bannerCopy.BAttr = getBlockedCreativetypesForAdThrive()
+
+			}
 			request.Imp[i].Banner = &bannerCopy
 			validImpExists = true
 		} else if request.Imp[i].Video != nil {
 			validImpExists = true
+			videoCopy := *request.Imp[i].Video
```

Comment:
Setting videoCopy is unnecessary unless the comparison on the following "if" evaluates to true.  Please move this line down into the "then" portion of the following "if". 

Likewise, the use of bannerCopy above could be similarly optimized, though it's a little more complicated because there are two cases where it might be needed.  Please fix that, too, assuming that a significant percentage of the time bannerCopy will not be changed.

Finally, unrelated to this Pull Request but FWIW:  is sufficient testing of the width and height values being done in the unchanged code related to a banner ad?  For example, are there no worries of only bannerCopy.W or bannerCopy.H being set without the other, nor worries of bannerCopy.Format[0] lacking one or both of W or H?

Reply:
Updated for videoCopy setting as you suggested. 
Using bannerCopy as such, as it is kind of validation section.

Reply:
Can you please elaborate on bannerCopy?  Its value is changed in only two cases:  (a) H and W are both null and Format is set, and (b) Publisher is adthrive.  I don't know but I would guess these two cases are more the exception than the rule.  If so, in the interest of performance, it is not desirable to copy a structure, make no change to it, and copy it back on a busy server.

Reply:
Copying the value to W and H is going to happen all the time in our case. Since our system not supporting oRTB2.5 for now. So need to create the bannerCopy whenever impression is of type banner.